packages/lib/determineBaseAppDirs.ts

```
@@ -1,12 +1,18 @@
 import { toSystemSlashes } from './path-utils';
+import { dirname, isAbsolute } from 'path';
 
 export default (profileFromArgs: string, appName: string, altInstanceId: string) => {
 	let profileDir = '';
 	let homeDir = '';
 
 	if (profileFromArgs) {
-		profileDir = profileFromArgs;
+		if (isAbsolute(profileFromArgs)) {
+			profileDir = profileFromArgs;
+		}
```

ESLint

[error] Closing curly brace does not appear on the same line as the subsequent block.

(brace-style)